src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -203,6 +226,20 @@ private function sanitizeContentTitle($title, $contentType)
         return $this->sanitizeUTF8Text($title);
     }
 
+    /**
+     * Try to fix the content retreived through the client-side rendering proxy.
```

Comment:
Could you explain why in the same comment?

Reply:
OK, fixed.